apps/condo/domains/organization/components/OrganizationSelect.tsx

```diff
@@ -78,8 +80,12 @@ export const InlineOrganizationSelect: React.FC = () => {
     const { setIsVisible: showCreateOrganizationModal, ModalForm: CreateOrganizationModalForm } = useCreateOrganizationModalForm({
         onFinish: async (createdOrganization) => {
             const organizationType = get(createdOrganization, 'type')
+            const isValidNextUrl = next && !Array.isArray(next) && isSafeUrl(next)
 
-            if (organizationType === MANAGING_COMPANY_TYPE) {
+            // After registration and created first organization we want redirect user to nextUrl
+            if (isValidNextUrl && !hasInvites && !link && !filteredEmployees.length) {
```

Comment:
- why !hasInvites blocks redirect to `next` ?

- why `!link` blocks redirect?

Reply:
We need to redirect to nextUrl only after registration and first creation of the organization. Therefore there is a check for lint, filteredEmployees, hasInvites, isValidNextUrl

Reply:
If user has invites/employees, then he is not be redirected to `next` link.
Are you sure that it is correct logic?
I think if the `next` parameter is specified, then the user should be redirected there.